spec/collections/data_sources/data_sources_nodes.yaml

```diff
@@ -0,0 +1,60 @@
+name: data_sources_nodes
+type: vertex
+delta: false
+
+indexes:
+  - type: persistent
+    fields: [type, ns]
+
+schema:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Data Source
+  description: |
+    A description of an external data source.
+    Data sources are external entities, usually governed and managed by an organization,
+    with data repositories, data policies, and specific data structures.
+    (e.g. taxonomy, ontology), individual data sources may provide data beyond this
+    general structure; the data source definition must provide a description
+    of those additional fields so that they can be utilized by interfaces.
+  type: object
+  additionalProperties: false
+  required:
+    [
```

Comment:
use the `- item` notation to be consistent

Reply:
Yes it was annoying to have the formatter add square brackets and then to have to manually change them back, but I did it :)

Reply:
Thanks! 😁